src/shared/meshes/cell_linked_list.hpp

```diff
@@ -96,7 +105,7 @@ void CellLinkedList::particle_for_split(const execution::SequencedPolicy &, cons
         // i_max = (M - m - 1) / 3 + 1, j_max = (N - n - 1) / 3 + 1
         // e.g. all_cells = (M,N) = (6, 9), (m, n) = (1, 1), then i_max = 2, j_max = 3
         const Arrayi all_cells_k = (all_cells_ - split_cell_index - Arrayi::Ones()) / 3 + Arrayi::Ones();
-        const size_t number_of_cells = get1DMeshSize(all_cells_k); // i_max * j_max
+        const size_t number_of_cells = get_1d_number_of_cells(all_cells_k); // i_max * j_max
```

Comment:
Seems to use "all_cell_k.prod()" better?
https://eigen.tuxfamily.org/dox/group__TutorialReductionsVisitorsBroadcasting.html

Reply:
Thank you for the suggestion! I have replaced the function with `prod()`